app/controllers/application_controller.rb

```diff
@@ -41,13 +40,15 @@ def auto_complete
     session[:auto_complete] = controller_name.to_sym
     respond_to do |format|
       format.any(:js, :html)   { render partial: 'auto_complete' }
-      format.json { render json: @auto_complete.inject({}){|h,a|
-        h[a.id] = a.respond_to?(:full_name) ? h(a.full_name) : h(a.name); h
-      }}
+      format.json do
+        render json: @auto_complete.inject({}){|h, a|
+                       h[a.id] = a.respond_to?(:full_name) ? h(a.full_name) : h(a.name); h
```

Comment:
Stop

Sent from my iPhone

On Jan 19, 2015, at 3:17 PM, "Hound" <notifications@github.com<mailto:notifications@github.com>> wrote:


In app/controllers/application_controller.rb<https://github.com/fatfreecrm/fat_free_crm/pull/408#discussion-diff-23187392>:

> @@ -41,13 +40,15 @@ def auto_complete
>      session[:auto_complete] = controller_name.to_sym
>      respond_to do |format|
>        format.any(:js, :html)   { render partial: 'auto_complete' }
> -      format.json { render json: @auto_complete.inject({}){|h,a|
> -        h[a.id] = a.respond_to?(:full_name) ? h(a.full_name) : h(a.name); h
> -      }}
> +      format.json do
> +        render json: @auto_complete.inject({}){|h, a|
> +                       h[a.id] = a.respond_to?(:full_name) ? h(a.full_name) : h(a.name); h


Line is too long. [90/80]
Do not use semicolons to terminate expressions.

—
Reply to this email directly or view it on GitHub<https://github.com/fatfreecrm/fat_free_crm/pull/408/files#r23187392>.